src/Indicator/PhpUnitTestCaseIndicator.php

```diff
@@ -62,7 +62,12 @@ public function isPhpUnitClass(Tokens $tokens, int $index): bool
     }
 
     /**
-     * @return \Generator array of [int start, int end] indices from sooner to later classes
+     * Returns an indices of PHPUnit classes in reverse appearance order.
+     * Order is important - it's reverted, so if we inject tokens into collection,
+     * we do it for bottom of file first, and then to the top of the file, so we
+     * mitigate risk of not visiting whole collcetion (final indices).
+     *
+     * @return \Generator array of [int start, int end] indices from later to earlier classes
```

Comment:
maybe update return type to `iterable` while we are here?

Reply:
```suggestion
     * @return iterable array of [int start, int end] indices from later to earlier classes
```